[DOCS] More doc updates to remove the CLI (#7874)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

</Prerequisites>

> There are slight differences between working with custom Expectations and those contributed to GX.

Reviewer: ```suggestion
Custom Expectations and those contributed to GX are different.
```
Why is this important? Why should a user care?

Author: because the way they are imported/referenced is different. There are tabs in the doc that describe the differences

<TabItem value="custom-expectations">

> If you're using a Custom Expectation you've built, you'll need to place it in the `great_expectations/plugins/expectations` folder of your Great Expectations deployment.

Reviewer: ```suggestion
If you're using a Custom Expectation you've created, add it to the `great_expectations/plugins/expectations` folder of your Great Expectations deployment.
```

When you

</TabItem>

<TabItem value="contrib-expectations">

> If you're using a Custom Expectation that is coming from the `Great Expectations Experimental` library, it will need to either be imported from there directly. To do this, we'll first need to `pip install great_expectations_experimental`.

Reviewer: ```suggestion
If you're using a Custom Expectation from the `Great Expectations Experimental` library, you'll need to import it.  Before you import it, run `pip install great_expectations_experimental`.
```

</TabItem>
</Tabs>

## Usage

> When you instantiate your <TechnicalTag tag="data_context" text="Data Context"/>, it will automatically make all plugins in the directory available for use,

Reviewer: ```suggestion
When you instantiate your <TechnicalTag tag="data_context" text="Data Context"/>, all plugins in the directory are automatically available, and this allows you to import your Custom Expectation from other locations.
```

allowing you to import your Custom Expectation from that directory whenever and wherever it will be used.
This import will be needed when an <TechnicalTag tag="expectation_suite" text="Expectation Suite"/> is created, *and* when a <TechnicalTag tag="checkpoint" text="Checkpoint"/> is defined and run.

<Tabs
  groupId="expectation-type"
  defaultValue='custom-expectations'
  values={[
  {label: 'Custom Expectations You\'ve Built', value:'custom-expectations'},
  {label: 'Custom Expectations Contributed To Great Expectations', value:'contrib-expectations'},
  ]}>

<TabItem value="custom-expectations">

```python
from expectations.expect_column_values_to_be_alphabetical import ExpectColumnValuesToBeAlphabetical
# ...
validator.expect_column_values_to_be_alphabetical(column="test")
```

</TabItem>

<TabItem value="contrib-expectations">

```python
from great_expectations_experimental.expectations.expect_column_values_to_be_alphabetical import ExpectColumnValuesToBeAlphabetical
# ...
validator.expect_column_values_to_be_alphabetical(column="test")
```

</TabItem>
</Tabs>

<div style={{"text-align":"center"}}>
<p style={{"color":"#8784FF","font-size":"1.4em"}}><b>

> Congratulations!<br/>&#127881; You've just used a custom expectation! &#127881;

Reviewer: ```suggestion

```
